src/ConsiliumTempus.Infrastructure/Persistence/Repository/AnnouncementRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using ConsiliumTempus.Application.Common.Interfaces.Persistence.Repository;
+using ConsiliumTempus.Domain.Announcement;
+using ConsiliumTempus.Infrastructure.Persistence.Database;
+
+namespace ConsiliumTempus.Infrastructure.Persistence.Repository;
+
+public class AnnouncementRepository(ConsiliumTempusDbContext dbContext) : IAnnouncementRepository
```

Comment:
Should have been a sealed class (this is will also solve the Architecture Tests failure, also ignore the unit tests, that's a false negative)